lib/node-detector.js

```diff
@@ -104,4 +105,16 @@ class NodeDetector {
   }
 }
 
+class NpmDetector {
+  static async detect () {
```

Comment:
any reason to use static?

Reply:
such a simple function does not require to be wrapped into a class as for my taste

Reply:
Agree. I considered which was proper...
Let me back to `general` here